lib/pxenforcer.js

```diff
@@ -55,6 +55,16 @@ class PxEnforcer {
             return cb();
         }
 
+        try {
+            //check if this is a telemetry command
```

Comment:
Can you add a space between the comment start and the first word? A standard that I'm following

Reply:
Also, it feels like a redundant note, how can `isTelemetryCommand` not explain it?

Reply:
removed